src/parser/paladin/retribution/modules/talents/EmpyreanPower.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import { formatDuration, formatNumber } from 'common/format';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+import ItemDamageDone from 'interface/ItemDamageDone';
+import Statistic from 'interface/statistics/Statistic';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { ApplyBuffEvent, DamageEvent, RemoveBuffEvent } from 'parser/core/Events';
+
+import { plotOneVariableBinomChart } from 'parser/shared/modules/helpers/Probability';
+
+import { EMPYREAN_POWER_CHANCE } from '../../constants';
+
+const BUFF_TIME: number = 15000 * .95;//add buffer since log events lmao
+const TRACK_BUFFER = 500;
+
+class EmpyreanPower extends Analyzer {
+
+  averageTimeTillBuffConsumed: number = 0;
+
+  hasProc: boolean = false;
+  procsWasted: number = 0;
```

Comment:
When setting a field to a default value that correctly implicitly types, you don't need to explicitly set the type. This might get linted out later, so it might potentially save you some time later on if the rule ever comes through.

```typescript
myField: number = 0; // Ok, but not needed
myField = 0; 

myField: boolean = false; // Ok, but not needed
myField = false;
```